core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1379,6 +1378,14 @@ public boolean convertPartitionNativeToParquet(long partitionTimestamp) {
                                     0
                             );
                         }
+                    } else {
+                        // no rows in column
```

Comment:
We should write default values in case of coltops for non-nullable columns.

Reply:
Compatibility test updated